azure-pipelines.yml

```diff
@@ -26,6 +26,5 @@ steps:
     displayName: 'npm run build'
 
   - script: |
-      yarn verify-api
-      yarn check-for-changed-files
```

Comment:
Can we leave the check-for-changed files and open an issue to reintroduce verify-api at least?

Reply:
I don't understand why either of these were removed. What am I missing?

Reply:
@FalseLobster Added the check-for-changed-files back.
@tkamireh The verify-api task doesn't exist and always fail.

Reply:
ah I see, you could add a line to scripts/just.config.js (it should just be following the update-api task definition)